parameter_files/WebParameters_2dii.yml

```diff
@@ -1,6 +1,6 @@
 default:
     paths:
-        project_location_ext: ./
+        project_location_ext: ../PACTA_analysis/
```

Comment:
Is there a particular reason to use the form `./` instead of the form `../PACTA_analysis/`?
The form "./" breakes the integration test. I could make it work but I'm trying to figure out 
if it's worth the effort.

Reply:
no, I think either should work.... `../PACTA_analysis/` is just more specific

Reply:
but, this is in the wrong PR, or?

Reply:
I don't know how this made it here but it is :)